src/components/empty-author-tile/EmptyAuthorTile.tsx

```diff
@@ -51,11 +52,13 @@ const Media = styled.div`
   padding: 32px 0 16px 0;
 `
 
-export interface EmptyAuthorTileProps {
-  onClick: () => void
-}
+const JoinUsLink = styled(ExternalLink)`
+  ${BtnLink}
+`
+
 
-export const EmptyAuthorTile = ({ onClick }: EmptyAuthorTileProps) => {
+export const EmptyAuthorTile = () => {
+  const discordUrl = getDiscordUrl()
```

Comment:
**I have here idea**

getDiscordUrl() repeats logic which is already done inside `gatsby-node.js`. We're loading all data connected with pages once, to avoid query repetition and because it takes less time than multiple queries.

To avoid `prop-drilling` here, you can use render-slot pattern from React. So to `EmptyAuthorTileProps` you can add property -> joinUs which will be type of `ReactNode`, so it allow us to pass any trigger button without messing this component API.